browser/ui/webui/settings/brave_default_extensions_handler.cc

```diff
@@ -76,3 +81,28 @@ void BraveDefaultExtensionsHandler::SetHangoutsEnabled(
         extensions::disable_reason::DisableReason::DISABLE_BLOCKED_BY_POLICY);
   }
 }
+
+void BraveDefaultExtensionsHandler::SetIPFSCompanionEnabled(
+    const base::ListValue* args) {
+  CHECK_EQ(args->GetSize(), 1U);
+  CHECK(profile_);
+  bool value;
+  args->GetBoolean(0, &value);
+
+  extensions::ExtensionService* service =
+    extensions::ExtensionSystem::Get(profile_)->extension_service();
+
+  if (value) {
```

Comment:
Nit: Really minor, but maybe `enabled` would be a better name for this variable?

Reply:
resolved by https://github.com/brave/brave-core/pull/2106/commits/ab5ca552427438c729b8564032fe51e2447c2c89